src/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+configure_file(${CMAKE_SOURCE_DIR}/cmake/bitcoin-config.h.in config/bitcoin-config.h @ONLY)
```

Comment:
This would generate the config file in `config/bitcoin-config.h` whereas with autotools it would be in `src/config/bitcoin-config.h`. I guess this is ok (as long as the proper include flags are passed)?

Reply:
It generates `${CMAKE_CURRENT_BINARY_DIR}/config/bitcoin-config.h`, which is equivalent to `${CMAKE_BINARY_DIR}/src//config/bitcoin-config.h`.

The related `include_directories()` goes two lines below :)